prometheus/src/common.tsx

```diff
@@ -148,7 +149,8 @@ export function GenericMetricsChart(props: {
                 }}
                 size="large"
               >
-                {refresh ? <Icon icon="mdi:pause" /> : <Icon icon="mdi:play" />}
+                {refresh ? <Tooltip title="Pause metrics"> <Icon icon="mdi:pause" /> </Tooltip>
+                  : <Tooltip title="Resume metrics"> <Icon icon="mdi:play" /> </Tooltip>}
```

Comment:
Please format this according to the style we use in Headlamp. I am not sure why the linter is not catching this, as the error in CI seems unrelated.

Reply:
fixed!